### PR TITLE
Fix/types

### PR DESCRIPTION
**Pull request by pgolovkin**
1. Fixed types for substrate
2. Added types troubleshooting script

### CODE REVIEW

File: query.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+from scalecodec.type_registry import load_type_registry_file
+from substrateinterface import SubstrateInterface
+
+substrate = SubstrateInterface(
+    url='wss://ws.framenode-8.s5.stg1.sora2.soramitsu.co.jp/',
+    type_registry_preset='default',
+    type_registry=load_type_registry_file('./custom_types.json'),
+)
+
+block_hash = substrate.get_block_hash(block_id=1103012)
```
Comment (modbrin) [inline query.py:11]: what this script does, test current types for decoding errors? why this block num?

**Review by modbrin — APPROVED** [review]

File: query.py (hunk 1 above)
Comment (modbrin) [reply query.py:11]: maybe this should be added to versioned types part, though this mechanism doesn't work in js

Comment (pgolovkin) [reply query.py:11]: With this script you can debug custom_types.json. If block parsing fails then you can run only this script for specific block and check for errors.
What types do you mean should be added with versioning?

Comment (modbrin) [reply query.py:11]: It seemed to me as if starting from specific block something broke, but the types look new, so no need for versioning here I guess

Comment (pgolovkin) [reply query.py:11]: got it. It works fine for new types. The problem might be if existing types are changing in a new version.

**Pull request merged**